### ISSUE TITLE
Strange behavior

### ISSUE THREAD

**Issue by Chilace** [Suppen/TediCross#36]
Sometimes a bot tries to process messages not in the right channels and produces an error message:
```
2017-11-21 15:57:22 [ERR]	Could not edit Telegram message: Error: No corresponding message ID for 382560129117323265
at MessageMap.getCorresponding (/app/lib/MessageMap.js:57:10)
at Promise.resolve.then (/app/lib/discord2telegram/setup.js:148:29)
at process._tickCallback (internal/process/next_tick.js:109:7)
2017-11-21 16:02:09 [ERR]	Could not edit Telegram message: Error: No corresponding message ID for 382546195597492234
at MessageMap.getCorresponding (/app/lib/MessageMap.js:57:10)
at Promise.resolve.then (/app/lib/discord2telegram/setup.js:148:29)
at process._tickCallback (internal/process/next_tick.js:109:7)
2017-11-21 16:06:44 [ERR]	Could not edit Telegram message: Error: No corresponding message ID for 382557993813671936
at MessageMap.getCorresponding (/app/lib/MessageMap.js:57:10)
at Promise.resolve.then (/app/lib/discord2telegram/setup.js:148:29)
at process._tickCallback (internal/process/next_tick.js:109:7)
2017-11-21 16:07:57 [ERR]	Could not edit Telegram message: Error: No corresponding message ID for 382546195597492234
at MessageMap.getCorresponding (/app/lib/MessageMap.js:57:10)
at Promise.resolve.then (/app/lib/discord2telegram/setup.js:148:29)
at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Comment by Suppen** [comment]
Interesting... I've actually forgotten to add both channel and server check to updates. As far as I can tell, it doesn't do anything else than give that error in the console.

I will fix this

**Comment by Suppen** [comment]
Fixed in commit 940a50eaeb5a50dc53184e4f0ab786827ff46aa4
